airflow/models.py

```diff
@@ -522,10 +522,12 @@ def collect_dags(
         Given a file path or a folder, this method looks for python modules,
         imports them and adds them to the dagbag collection.
 
-        Note that if a .airflowignore file is found while processing,
-        the directory, it will behaves much like a .gitignore does,
+        Note that if a ``.airflowignore`` file is found while processing
```

Comment:
Where in the file system does this .airflowignore need to be? Why not have a placeholder (empty) .airflowignore committed or part of the distribution, so folks know to override it in place?

Reply:
This .airflowignore file should be in DAG_FOLDER. I mentioned it in my change in “docs/concepts”. 

It’s a great idea to have a placeholder .airflowignore. But it’s a bit tricky to do this since users normally need to create “dags” folder by themselves after initdb.

Reply:
I didn’t know the existence of this nice feature either previously. that’s exactly why we need to mention it to users more explicitly ;-)